README.md

```diff
@@ -27,27 +27,88 @@ cd ./devop-tools/docker/data-source-services && docker-compose up --build -d
 ```
 
 #### Specific Setup - Windows
+
 * Kill, and optionally disable, the `World Wide Web Publishing Service`
 * Run `set COMPOSE_CONVERT_WINDOWS_PATHS=1` in your CMD or PowerShell terminal
 * Optionally, [read](https://github.com/docker/compose/issues/4303#issuecomment-379563170) this bug report.
 
 ### Optional: Data Source Tools
+
 * Optionally included is the following tools:
-  * phpMyAdmin
+* phpMyAdmin
 * `cd ./devop-tools/docker/data-source-tools && docker-compose up --build -d`
 
 ## phpMyAdmin
+
 If the optional tools are launched, you can find phpMyAdmin at: localhost:8080
+
 * It supports the following databases...
-  * mariadb106 (lts)
-  * mariadb1011 (lts)
+* mariadb106 (lts)
+* mariadb1011 (lts)
 
 ## Scripts
+
 Inside the `scripts` folder you will find one-off scripts to help with tasks.
 
 * `db_migrate.sh` - Helps migrate databases between versions of mysql.
 
+## CLI Tooling Setup
+
+Add the `bin` directory to your path to use **Hop** and **Git Scripts**.
+
+## Hop
+
+MISSING DESCRIPTION
```

Comment:
This is what shows when you run the command, not sure if someone wants to write a more details description here.

```
A script that makes it easy to hop into your project containers and run commands.
```